### DISCUSSION TITLE
Light Curve Filters

### DISCUSSION THREAD

**Discussion by nbungart1999** [phoebe-project/phoebe2#454] [Questions]
When making the datasets for light curves, how do I make each dataset a different filter such as the B, V, and TESS:T filters and then plot all 3 on the same graph or separate graphs?

**Comment by aprsa** [comment]
You need to add a dataset for each, i.e.:
```
import phoebe
b = phoebe.default_binary()
b['teff@secondary'] = 5000
b.add_dataset('lc', dataset='lcB', passband='Johnson:B', times=phoebe.linspace(0, 1, 51))
b.add_dataset('lc', dataset='lcV', passband='Johnson:V', times=phoebe.linspace(0, 1, 51))
b.add_dataset('lc', dataset='lcR', passband='Johnson:R', times=phoebe.linspace(0, 1, 51))
b.run_compute()
b.plot(show=True)
```

**Reply by nbungart1999** [reply]
Thanks! I knew I needed to make 3 different datasets but I didn't know what "description" to put in the dataset to specify which band I wanted.

**Reply by nbungart1999** [reply]
Also in your example, you have the BVR filters so for my TESS:T filter do I just put that in the same spot, or is there some other notation for it?

**Reply by gecheline** [reply]
It's the same, just replace 'Johnson:x' with 'TESS:T'

**Comment by gecheline** [comment]
I'll just add, if you want to see which light curve corresponds to each dataset in the same plot, you need to add a legend to the plot:

`b.plot(legend=True, show=True)`

and if you want to plot each one separately, you should filter on the corresponding dataset first:

`b.filter(dataset='lcB').plot(show=True)`

**Reply by nbungart1999** [reply]
That's really helpful thanks!
